pkg/server/status.go

```diff
@@ -599,6 +600,35 @@ func (s *statusServer) Details(
 	return resp, nil
 }
 
+func (s *statusServer) SystemDetails(
```

Comment:
Would it make sense to fold this into

https://github.com/cockroachdb/cockroach/blob/eb38f2002e2dd275dd4e47c0b96c0405b655f71c/pkg/server/status.go#L554-L558

instead?

Also, the output of `uname -r` would also be interesting because we might be able to parse it automatically in the future to warn about vulnerable kernel versions.

It would also be better to not fail this request on non-linux systems but to return an empty value there (and possibly nonempty ones in the future).